packages/story-editor/src/components/reorderable/reorderableItem.js

```diff
@@ -36,9 +36,9 @@ function ReorderableItem({
   disabled,
   ...props
 }) {
-  const {
-    actions: { handleStartReordering },
-  } = useReorderable();
+  const handleStartReordering = useReorderable(
+    (v) => v.actions.handleStartReordering
```

Comment:
What does v stand for everywhere? What‘s wrong with `state`?

Reply:
`v` is just shorthand for `value`. I think it makes more sense than `state` cus a context holds a `value` which can have refs, state, actions, or an incoming config prop. Though I agree that `value` can seem a little too vague, I think there's a reason that react makes providers hold and expose a `value` prop and I'm following that convention. I'm assuming the library author is as well with the `v` shorthand.

Also have never liked `state.<state | actions>...` and `value.<state | actions>` conflates the term state less.

Anyway, that's my rationale. At the end of the day it's just a name, so I'm fine with whatever. Just thought I'd pitch it as I propose a pattern where we might be updating the use of it.

Reply:
Makes sense - maybe we write `value` instead of `v` just for readability? Small nit but that way we don't conflate `state` too much too.

Reply:
> Anyway, that's my rationale. At the end of the day it's just a name, so I'm fine with whatever. Just thought I'd pitch it as I propose a pattern where we might be updating the use of it.

@littlemilkstudio Makes sense replacing `state.state.foo` with `value.state.foo` but wondering why is `useReorderable((value)) => value.actions...` better than `useReorderable(({ actions })) => actions...`?